packages/ui-react/src/containers/AccountModal/forms/Login.tsx

```diff
@@ -34,8 +36,8 @@ const Login = () => {
     }),
     onSubmit: ({ email, password }) => accountController.login(email, password, window.location.href),
     onSubmitSuccess: () => {
+      accessController.generatePassport();
```

Comment:
I think this would be better placed in the accountController login method. Logic in the components should be limited to UI flows.

Reply:
But aren't we against using controllers inside other controllers? In that case I will need to use the access service here and make sure the needed params are provided.